cmd/module.go

```
@@ -55,8 +56,13 @@ var getModulesCmd = &cobra.Command{
 			logs.Logger.Error(err)
 		}
 		if modules != nil {
-			fmt.Print(string(modules))
+			output, rerr := json.Marshal(modules)
+			if rerr != nil {
```

This error is not returned. Why?